docs/content/local-file.md

```diff
@@ -132,6 +134,23 @@ is determined by your build server, and `.nupkg`s created locally have zero
 version. This is currently supported only for [source
 override](#Source-override).
 
+Another use case is testing an upgraded pre-release package with an incremented 
+version, against a consuming project.  In this case you want paket in the consuming 
+project to fetch a higher version than the paket.local file indicates.
+
+Example `paket.local` file with version override
+
+```paket
+    // Argu is searched in specific version, grab a specific version
+    nuget Argu -> source C:\github\Argu\bin version 1.0.0
+```
+
+>Notes
+>
+>* `version` is currently supported only for [Source override](#Source-override).
+>* The syntax allows only a specific version, **not** inequality expressions like >= .
```

Comment:
Instead `inequality expressions like >=` let's have an already used term for that: `version constraint` http://fsprojects.github.io/Paket/nuget-dependencies.html#Version-constraints